pandas/tests/scalar/timestamp/test_arithmetic.py

```diff
@@ -66,6 +66,17 @@ def test_delta_preserve_nanos(self):
         result = val + timedelta(1)
         assert result.nanosecond == val.nanosecond
 
+    def test_rsub_dtscalars(self, tz_naive_fixture):
+        # In particular, check that datetime64 - Timestamp works GH#28286
+        td = Timedelta(1235345642000)
+        ts = Timestamp.now(tz_naive_fixture)
+        other = ts + td
+
+        assert other - ts == td
+        assert other.to_pydatetime() - ts == td
```

Comment:
Is anything above this line not tested elsewhere? Just was a little counter intuitive to read as I thought this PR was only about datetime64 obects

Reply:
trying to follow the pattern in these tests of testing all datetime-like scalars in the same place

Reply:
Makes sense